tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -0,0 +1,287 @@
+readonly version_file="${repo_root_dir}/VERSION"
+readonly versions_yaml="${repo_root_dir}/versions.yaml"
+
+readonly clh_builder="${repo_root_dir}/tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh"
+readonly firecracker_builder="${repo_root_dir}/tools/packaging/static-build/firecracker/build-static-firecracker.sh"
+readonly kernel_builder="${repo_root_dir}/tools/packaging/kernel/build-kernel.sh"
+readonly qemu_builder="${repo_root_dir}/tools/packaging/static-build/qemu/build-static-qemu.sh"
+readonly rootfs_builder="${repo_root_dir}/tools/packaging/guest-image/build_image.sh"
+
+workdir="${WORKDIR:-$PWD}"
+
+destdir="${workdir}/kata-static"
+
+die() {
+	msg="$*"
+	echo "ERROR: ${msg}" >&2
+	exit 1
+}
+
+info() {
+	echo "INFO: $*"
+}
+
+error() {
+	echo "ERROR: $*"
+}
+
+usage() {
+	return_code=${1:-0}
+	cat <<EOT
+This script is used as part of the ${project} release process.
+It is used to create a tarball with static binaries.
+
+
+Usage:
+${script_name} <options> [version]
+
+Args:
+version: The kata version that will be use to create the tarball
+
+options:
+
+-h|--help      	      : Show this help
+--build=<asset>       :
+	all
+	cloud-hypervisor
+	firecracker
+	kernel
+	qemu
+	rootfs-image
+	rootfs-initrd
+	shim-v2
+EOT
+
+	exit "${return_code}"
+}
+
+#Install guest image
+install_image() {
+	info "Create image"
+	"${rootfs_builder}" --imagetype=image --prefix="${prefix}" --destdir="${destdir}"
+}
+
+#Install guest initrd
+install_initrd() {
+	info "Create initrd"
+	"${rootfs_builder}" --imagetype=initrd --prefix="${prefix}" --destdir="${destdir}"
+}
+
+#Install kernel asset
+install_kernel() {
+	info "build kernel"
+	export kernel_version=="$(yq r $versions_yaml assets.version)"
+	"${kernel_builder}" setup
+	"${kernel_builder}" build
+	info "install kernel"
+	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" install
+}
+
+#Install experimental kernel asset
+install_experimental_kernel() {
+	info "build experimental kernel"
+	export kernel_version="$(yq r $versions_yaml assets.kernel-experimental.version)"
+	"${kernel_builder}" -e setup
+	"${kernel_builder}" -e build
+	info "install experimental kernel"
+	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" -e install
+}
+
+# Install static qemu asset
+install_qemu() {
+	info "build static qemu"
+	export qemu_repo="$(yq r $versions_yaml assets.hypervisor.qemu.url)"
+	export qemu_version="$(yq r $versions_yaml assets.hypervisor.qemu.version)"
+	"${qemu_builder}"
+	tar xvf "${builddir}/kata-static-qemu.tar.gz" -C "${destdir}"
+}
+
+# Install static firecracker asset
+install_firecracker() {
+	info "build static firecracker"
+	"${firecracker_builder}"
+	info "Install static firecracker"
+	mkdir -p "${destdir}/opt/kata/bin/"
+	sudo install -D --owner root --group root --mode 0744 firecracker/firecracker-static "${destdir}/opt/kata/bin/firecracker"
+	sudo install -D --owner root --group root --mode 0744 firecracker/jailer-static "${destdir}/opt/kata/bin/jailer"
```

Comment:
Shouldn't this perm be `0644`?

Reply:
No,  this is a binary that kata may use it requires exec permissions.